ansible/playbooks/tasks/azure-cluster-hana.yaml

```diff
@@ -12,7 +12,7 @@
     hana_resource_clone: "{{ hana_crm_mon_xml.stdout | regex_search('ms msl_SAPHana_') }}"
     hana_resource: "{{ hana_crm_mon_xml.stdout | regex_search('primitive rsc_SAPHana_') }}"
     ip_resource: "{{ hana_crm_mon_xml.stdout | regex_search('primitive rsc_ip_') }}"
-    ip_nc: "{{ hana_crm_mon_xml.stdout | regex_search('primitive rsc_nc_') }}"
+    ip_nc: "{{ hana_crm_mon_xml.stdout | regex_search('primitive rsc_socat_') }}"
```

Comment:
What is this fix about?

Reply:
I changed to resource name from nc to socat on an earlier PR, but I didn't update this regex that searched for the old name.

I changed the name as [Azure uses socat rather than nc](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#create-sap-hana-cluster-resources).  Naming it nc is now misleading.